project_event/models/project_task.py

```diff
@@ -139,6 +139,12 @@ class Task(models.Model):
         default=False,
     )
     color = fields.Char(related='category_id.color')
+    date_start = fields.Datetime(
```

Comment:
@rimbendhaou : Define two times "date_start" in the same model . Beside no need to redefine  date_end in project.task for this issue.